documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/action-parameters.md

```diff
@@ -7,7 +7,7 @@ permalink: /documentation/working-with-testcafe-studio/recording-tests/test-acti
 
 ## Element Selector
 
-Most of the on-page actions have the *Element Selector* parameter that specifies a target element. This parameter accepts a [Selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/) function.
+On-page actions have the *Element Selector* parameter that specifies a target element. This parameter accepts a [Selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/) function.
```

Comment:
@dirk-pieterse 
Not all the on-page actions have this parameter.

Reply:
Ok, but 'Most of' is vague. Which actions?

Reply:
10 out of 12. Do we need to name them? The details are in the actions' descriptions.

Reply:
Some on-page actions have... (some can be a link)

Reply:
I'll rewrite in another way:  "The *Element Selector* parameter specifies a target element for an on-page test action.

Reply:
Ok